embedchain/utils.py

```diff
@@ -2,9 +2,38 @@
 
 
 def clean_string(text):
+    """
+    This function takes in a string and performs a series of text cleaning operations. 
+
+    Args:
+        text (str): The text to be cleaned. This is expected to be a string.
+
+    Returns:
+        cleaned_text (str): The cleaned text after all the cleaning operations have been performed.
+    """
+    # Replacement of newline characters:
+    # This replaces all newline characters '\n' in the text with a space character ' '.
```

Comment:
Hey, we can keep a single line.
like here,
# replace of newline characters

the other line can be omitted.